#gz_robot_x | gz_robot_y | at_goal_a [0,1] | dist_from_goal_a | at_goal_b | dist_from_goal_b
float64 gz_robot_x
float64 gz_robot_y 
bool at_goal_a
bool at_goal_b
float64 dist_from_goal_a
float64 dist_from_goal_b
time data
#future: collision